### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Romanskyy**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,13 +1,30 @@
...
-    pass
+    return [
+        {
+            "errors": [
+                format_linter_error(error)
+                for error in errors if len(errors) > 0
+            ],
+            "path": file_path,
+            "status": "failed" if len(errors) > 0 else "passed",
+        }
```
Comment (NazarSheremet) [inline app/main.py]: Use format_single_linter_file function which does the same

File: app/main.py [hunk 2]
```diff
@@ -1,13 +1,30 @@
...
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "errors": [format_linter_error(error) for error in errors],
+        "path": file_path,
+        "status": "failed",
```
Comment (NazarSheremet) [inline app/main.py]: Need to add a check of status, if the status is passed or failed

**Review by NazarSheremet — CHANGES_REQUESTED** [review]
Need some changes

File: app/main.py (hunk 1 above)
Comment (Romanskyy) [reply app/main.py]: done

File: app/main.py (hunk 2 above)
Comment (Romanskyy) [reply app/main.py]: done

**Comment by Romanskyy** [comment]
please, review my PR

**Review by NazarSheremet — APPROVED** [review]
Good job!

**Review by YuriiSmolii — APPROVED** [review]
LGTM

**Pull request closed without merging**